mods/tuxemon/db/locale/cs_CZ.json

```diff
@@ -5,7 +5,6 @@
     "combat_used_x_on_y": "${{user}} použil ${{name}} na ${{target}}!",
     "combat_call_tuxemon": "Běž ${{name}}!",
     "combat_opponent_call_tuxemon": "${{user}} posílá ven ${{name}}!",
-    "combat_state_poison_damage": "${{name}} byl otráven!",
```

Comment:
These haven't been used for years (they were before the po files) you don't need to update them.

Create a ticket if you like.